### ISSUE TITLE
a11y - no focus state for primary action

### ISSUE THREAD

**Issue by robdodson** [addyosmani/smaller-pictures-app#2]
I don't think there's currently a focusable element which will allow me to upload a photo. May want to add tabindex=0 and a keyboard handler to whichever element triggers the upload
